linegraph.lua

```diff
@@ -1,70 +1,80 @@
-tfm.exec.addPhysicObject(-1,400,-600,{type=0,width=10,height=10,foreground=true,friction=0.3,restitution=0,dynamic=false,miceCollision=true,groundCollision=true})
-
-function getMin(tbl)
-  local min = tbl[1]
-  for k, v in ipairs(tbl) do
-    if v < min then
-      min = v
-    end
-  end
-  return min
-end
-
-function getMax(tbl)
-  local max = tbl[1]
-  for k, v in ipairs(tbl) do
-    if v > max then
-      max = v
-    end
-  end
-  return max
-end
-
-function map(tbl, f)
-  local res = {}
-  for k, v in pairs(tbl) do
-    res[k] = f(v)
-  end
-  return res
-end
-
-function range(from, to, step)
-	res = {}
-	for i = from, to, step do table.insert(res, i) end
+eventNewGame = function()
```

Comment:
Can you describe why did you insert this line under new Game event?  This will not executed if the player override this in their code again (correct me if I'm wrong)

Reply:
OK! But what happens if the programmer override eventNewGame back in their code?